group-generators/generators/aavechan-above10/index.ts

```diff
@@ -0,0 +1,44 @@
+import axios from "axios";
+import { Tags, ValueType, GroupWithData, FetchedData } from "topics/group";
+import {
+  GenerationContext,
+  GenerationFrequency,
+  GroupGenerator,
+} from "topics/group-generator";
+
+const generator: GroupGenerator = {
+  generationFrequency: GenerationFrequency.Daily,
+
+  generate: async (context: GenerationContext): Promise<GroupWithData[]> => {
+    const url =
+      "https://api.flipsidecrypto.com/api/v2/queries/51f179ec-4200-4070-ba05-a8c2e12cd68c/data/latest";
+    const response = await axios.get(url);
+    const allDelegatees = response.data;
+    const sismoData: FetchedData = {};
+
+    for (const delegatee of allDelegatees) {
+      if (delegatee.AMOUNT >= 10) {
+        sismoData[delegatee.DELEGATING_ADDRESS] = 1;
+      }
+    }
+
+    //add test address to aavechan list for testing
```

Comment:
Can you remove this test address?
When you use Sismo Connect you will see that you can enable a developer mode that allow you to put test addresses to test your Sismo Connect flow ;)